### DISCUSSION TITLE
New feature to add padding

### DISCUSSION THREAD

**Discussion by churdman001** [rxhanson/RectanglePro-Community#297] [Ideas]
Some monitors make it difficult to see the pixels right next to the monitor's bezel. I propose adding a feature that allows adding padding along the outside edges of the screen.

**Comment by rxhanson** [comment]
I've been meaning to add this into the UI for several releases now, and somehow it keeps getting pushed back. I'll see if I can roll it into the next one (which is a little ways off at the moment).

Until then, here are the terminal commands for it:

```
defaults write com.knollsoft.Hookshot screenEdgeGapTop -int 10
defaults write com.knollsoft.Hookshot screenEdgeGapBottom -int 10
defaults write com.knollsoft.Hookshot screenEdgeGapLeft -int 10
defaults write com.knollsoft.Hookshot screenEdgeGapRight -int 10
```

You'll want to replace the 10 with whatever value you want. Restart the app afterwards.

I do have plans to make a lot of settings in the app display specific, but that's further down the road.

As an aside, you can use nearly all of the Terminal commands from Rectangle (https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md), just replace com.knollsoft.Rectangle with com.knollsoft.Hookshot.
